docs/software/camera.md

```diff
@@ -116,7 +116,7 @@ from any RPi camera, USB camera or an IP camera network stream.
     motionEye can be updated to the latest version via
 
     ```sh
-    sudo pip2 install -U motioneye
+    sudo pip3 install -U motioneye
```

Comment:
Since motionEye v0.43.1b1 is currently a pre-release, this is actually the way to do it.
```suggestion
    sudo pip3 install -U --pre motioneye
```